app/models/city.rb

```diff
@@ -1,6 +1,12 @@
 class City < ApplicationRecord
   belongs_to :country
   has_many :airports
+  has_many :departure_flights, through: :airports
+  has_many :arrival_flights, through: :airports
 
   validates :name, presence: true
+
+  def self.all_cities_with_country_symbol
+    joins(:country).select('cities.id', 'cities.name', 'countries.symbol as country_symbol')
+  end
```

Comment:
I thnk you might want to use `left_join` instead of `join`, which defaults to an inner join

Reply:
done